src/App.css

```diff
@@ -1,39 +1,46 @@
-.App {
-  text-align: center;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.container {
+  padding: 2rem;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.resultBar {
+  height: 6rem;
+  width: 100%;
+  font-size: 3rem;
+  text-align: right;
+  padding: 0 1rem;
+  background-color: gray;
+  color: #fff;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.keypad {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  width: 100%;
 }
 
-.App-link {
-  color: #61dafb;
+.keypad :nth-child(17) {
+  grid-column: span 2;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.keypad :nth-child(4),
+.keypad :nth-child(8),
+.keypad :nth-child(12),
+.keypad :nth-child(16),
+.keypad :nth-child(19) {
+  background-color: #ffb05c;
+}
```

Comment:
- [x] Please make sure to use the correct color for the operators' buttons background `#F5913E` to follow the wireframe design.